docs/proposals/tinyms-support.md

```diff
@@ -0,0 +1,139 @@
+#  Implement Sedna Python SDK support TinyMS
+# 1. Project Background
```

Comment:
The headline should follow the existing proposal template, such as:
- Motivation
- Goals
- ...